oak/server/invocation.cc

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "oak/server/invocation.h"
+
+#include "absl/memory/memory.h"
+#include "asylo/util/logging.h"
+
+namespace oak {
+
+std::unique_ptr<Invocation> Invocation::Create(MessageChannelReadHalf* invocation_channel) {
```

Comment:
The semantics of this method is more like "receive" than "create", but not sure if it makes sense to rename. Up to you.